.github/workflows/continuous-delivery.yml

```diff
@@ -0,0 +1,42 @@
+# Each pr get's its own environment that gets cleaned up if the deploy is successful
```

Comment:
Let's add a description to clarify why CI and CD files are defined distinctly.

After reading the two files it appears that they have different triggers and target different code promotion/environment flows.